helper/cert/source_gen.go

```diff
@@ -17,8 +17,19 @@ import (
 	"strings"
 	"sync"
 	"time"
+
+	"github.com/hashicorp/vault-k8s/leader"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	informerv1 "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/kubernetes"
 )
 
+// Name of the k8s Secret used to share the caBundle between leader and
+// followers
+const certSecretName = "vault-injector-certs"
```

Comment:
This being fixed assumes a maximum of 1 injector deployment per namespace right? Seems reasonable to me

Reply:
There can only be 1 injector deployment because the mutating webhook annotation prefix `vault.hashicorp.com` is not configurable.